.ci_support/osx_.yaml

```diff
@@ -0,0 +1,10 @@
+MACOSX_DEPLOYMENT_TARGET:
+- '10.9'
```

Comment:
should we use a more up to date version here?

Reply:
yes, currently the windows build is failing because my fix for H3 only built static libraries (not shared libraries) and we need the h3.dll to install h3-py. working on a fix over on the H3 feedstock.

Reply:
yes, currently the windows build is failing because my fix for H3 only built static libraries (not shared libraries) and we need the h3.dll to install h3-py. working on a fix over on the H3 feedstock.